assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -3,10 +3,51 @@ import { FrontendRichText } from '../../components/FrontendRichText/FrontendRich
 
 const { __ } = wp.i18n;
 
+const readCookie = (name) => {
+  const nameEQ = name + '=';
+  const ca = document.cookie.split(';');
+  let c;
+  for (let i = 0; i < ca.length; i++) {
+    c = ca[i];
+    while (c.charAt(0) === ' ') {
```

Comment:
We could use `trimStart` here 🙂

Reply:
Not sure if it's needed, but we can safely use `trim` here. I also simplified the rest of the function.